packages/safe-tools-client/app/css/schedule.css

```diff
@@ -59,7 +59,26 @@
 }
 
 .temporary-form-container {
-  display:flex;
-  justify-content:center;
-  margin-top:var(--boxel-sp-xl);
-}
\ No newline at end of file
+  display: flex;
+  justify-content: center;
+  margin-top: var(--boxel-sp-xl);
+}
+
+.safe-tools__dashboard-network-selector {
+  display: inline-flex;
+  align-items: center;
+}
+
+.safe-tools__dashboard-network-selector.boxel-select .boxel-select__item {
+  font-weight: 600;
+  font-size: var(--boxel-font-size-sm);
+}
+
+.safe-tools__dashboard-network-selector .ember-power-select-status-icon {
+  background: url('/@cardstack/boxel/images/icons/caret-down-cyan.svg')
+    no-repeat;
+  width: 11px;
+  height: 9px;
+  margin-top: 5px;
+  display: inline-block;
+}
```

Comment:
Locally I was able to get this working with unmodified `caret-down` thanks to [this article](https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images):

<img width="508" alt="Cardstack Safe Tools 2022-10-26 12-38-24" src="https://user-images.githubusercontent.com/43280/198097433-9c76526d-ed4a-4503-a71e-ac5db36c603a.png">

```
  background-color: var(--boxel-cyan);
  -webkit-mask-image: url('/@cardstack/boxel/images/icons/caret-down.svg');
  -webkit-mask-repeat: no-repeat;
  mask-image: url('/@cardstack/boxel/images/icons/caret-down.svg');
  mask-repeat: no-repeat;
```

Reply:
🤦‍♀️  I didn't use the --webkit prefix, I'll try to apply it

Reply:
ya… I hate the prefix mess, people sometimes use CSS preprocessing for this but we have yet to enter that realm 😯